packages/cadl-ranch-specs/http/azure/core/lro/rpc-legacy/mockapi.ts

```diff
@@ -25,3 +25,44 @@ Scenarios.Azure_Core_Lro_Rpc_Legacy_CreateResourcePollViaOperationLocation = pas
     return { status: 200, body: json(response) };
   }),
 ]);
+
+const expectedModel = { modelId: "123", description: "hello" };
+let createPollResourceLocationCount = 0;
+
+Scenarios.Azure_Core_Lro_Rpc_Legacy_CreateResourcePollViaOperationLocationAndResourceLocation = passOnSuccess([
+  mockapi.post(
+    "/azure/core/lro/rpc/legacy/create-resource-poll-via-operation-location-final-result-in-resource-location/documentModels:build",
+    (req) => {
+      req.expect.containsQueryParam("api-version", "2022-12-01-preview");
+      req.expect.bodyEquals(expectedModel);
+      createPollResourceLocationCount = 0;
+      return {
+        status: 202,
+        headers: {
+          "operation-location": `${req.baseUrl}/azure/core/lro/rpc/legacy/create-resource-poll-via-operation-location-final-result-in-resource-location/operations/1234567890?api-version=2022-12-01-preview`,
+        },
+      };
+    },
+  ),
+  mockapi.get(
+    "/azure/core/lro/rpc/legacy/create-resource-poll-via-operation-location-final-result-in-resource-location/operations/1234567890",
+    (req) => {
+      req.expect.containsQueryParam("api-version", "2022-12-01-preview");
```

Comment:
nit, does python support this?

Reply:
Yeah, at least now I have work around to pass it